Fulfillment/UserInterface/01a_List.md

```diff
@@ -0,0 +1,522 @@
+
+The right side of the *Attributes* tab displays all attributes that are assigned to the selected group in the left margin column. Depending on the attribute group selected, the fields displayed vary. The dispatch note attributes are imported via the fulfiller driver. All fields are locked and, therefore, read-only.
+
+- ![Folders](../../Assets/Icons/Folders01.png "[Folders]") (Folders)  
+    Attribute group that contains attribute sub-groups. Click the attribute group or the arrow *>* left to the attribute group to unfold the group and display the attribute sub-groups.
+
+- ![Folder](../../Assets/Icons/Folder01.png "[Folder]") (Folder)  
+    Attribute group. Click the attribute group to display all attributes that are assigned to the selected attribute group on the right side of the *Attributes* tab.
+
+[comment]: <> (HG: Diese Info ist nicht im Omni-Channel. Hinzufügen dort oder hier wegglassen?)
+
+- ![Fade in/out](../../Assets/Icons/FadeInOut01.png "[Fade in/out]") (Fade in/out)    
+    Click this button to hide or display the left margin column with the attribute groups. When the left margin is displayed and you click this button, the column is hidden. When the column is hidden and you click this button, the column is displayed again.
+
+
+## Dispatch note "Dispatch note ID" &ndash; Dependencies
+
+*Fulfillment > Dispatch notes > Tab LIST > Select a dispatch note > Tab Dependencies*
+
+![Dependencies](../../Assets/Screenshots/Fulfillment/DispatchNotes/DispatchNoteDependencies.png "[Dependencies]")
+
+**Dependencies**
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+    Click this button to update the list of dispatch note dependencies.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+    Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- [x]     
+    Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all dependent entities in the list are selected.
+
+The list displays all dependencies of the selected dispatch note. Depending on the settings, the displayed columns may vary. All fields are read-only. 
+
+- *Dependent entity ID*   
+    Identification number of the dependent entity. The ID number is automatically assigned by the system. 
+
+- *Dependent entity type*   
+    Type of the dependent entity. 
+
+- *Change tracking mode*   
+    Change tracking mode (ETL mode) of the dependent entity. The following options are available:
+    - **Manual**
+    - **Semi-automatic**
+    - **Semi-automatic, changes must be confirmed by another user**   
+    - **Automatic**
+
+- *Dependent entity friendly identifier*   
+    Descriptive name of the dependent entity. The friendly identifier is normally used to search for the entity in the user interface, for example, a reference number or an SKU.
```

Comment:
Sollten uns abstimmen- in PIM und OC steht es jetzt so:
"Further, more descriptive identifier of the dependent entity, for instance the SKU number or a bill number."